cli/cmd/plugin/ui/web/tanzu-ui/src/shared/models/ClusterClass.ts

```diff
@@ -14,11 +15,42 @@ export interface ClusterClassVariable {
     required?: boolean,
 }
 
+export interface CCDefinition {
+    name: string,
+    variables: CCVariable[],
+    requiredVariables: () => CCVariable[],
+    basicVariables: () => CCVariable[],
+    intermediateVariables: () => CCVariable[],
+    advancedVariables: () => CCVariable[],
+}
+
+export interface CCVariable {
+    name: string,
+    taxonomy: ClusterClassVariableType,             // field classified according to known types
+    uiClassification: CCUiClassification, // field categorized as where it belongs in the UI
+    description?: string,
+    default?: any,
+    required?: boolean,
+    possibleValues?: string[],
+    children?: CCVariable[],
+}
+
+// for some reason, eslint is reporting these enum values as unused
```

Comment:
I think this may be caused by us setting 'no-unused-vars' in the eslintConfig in package.json. Can you try changing it to '@typescript-eslint/no-unused-vars'

Reply:
The `no-unused-vars` comment works as is; my "for some reason" comment is just lamenting that eslint bothers us with an unused enum